src/rascal/representations/calculator_spherical_expansion.hh

```diff
@@ -1636,8 +1636,15 @@ namespace rascal {
         AdaptorTraits::NeighbourListType::half};
     using math::PI;
     using math::pow;
-
     constexpr bool ExcludeGhosts{true};
+    constexpr static size_t ClusterLayer{
+        StructureManager::template cluster_layer_from_order<2>()};
+    const bool are_some_centers_masked{manager->are_some_centers_masked()};
```

Comment:
It would make more sense to name this `are_any_centers_masked()`, since that corresponds more closely to the underlying logical operation